custom/conf/app.ini.sample

```diff
@@ -286,10 +286,12 @@ LOG_SQL = true
 DB_RETRIES = 10
 ; Backoff time per DB retry (time.Duration)
 DB_RETRY_BACKOFF = 3s
-; Max idle database connections on connnection pool, default is 0
+; Max idle database connections on connnection pool, default is 2 or 0 on mysql
 MAX_IDLE_CONNS = 0
```

Comment:
I think we should change this default, even for MySQL/MariaDB. From your comment in the originating issue I get that we now have the two necessary settings for that (`MAX_IDLE_CONNS` and `CONN_MAX_LIFETIME`). The minimum value to hold should be the number of connections needed for serving a typically "big" http request.

By typically "big" I mean we should find one particular Gitea operation that is prone to consume several connections and choose a reasonable average value for it (e.g. posting a comment makes several queries outside the insert transaction, plus the actual insert).

Reply:
I prefer a break change to default value as 2

Reply:
```suggestion
MAX_IDLE_CONNS = 2
```

Reply:
Do you think we should set a default Max lifetime? I recall there were issues that MYSQL users were having with idle connections being closed under their feet.